core/libp2p/host.cpp

```diff
@@ -5,11 +5,26 @@
 
 #include "libp2p/host.hpp"
 
+#include <string_view>
+
+namespace {
+  constexpr std::string_view kLibp2pVersion = "ipfs/0.1.0";
+  constexpr std::string_view kClientVersion = "cpp-libp2p/0.1.0";
```

Comment:
They are such in Go implementation as well. Probably, this is because they are not Multi-something, but just versions, so there's no convention about slashes for them